Packs/Lumu/Integrations/Lumu/Lumu.py

```diff
@@ -0,0 +1,1080 @@
+        new_comment = add_prefix_to_comment(original_commnet)
+        key = client.api_key
+        try:
+            if reason := parsed_args.delta.get("closeReason"):
+                notes = parsed_args.delta.get("closeNotes", "")
+                user = parsed_args.delta.get("closingUserId", "")
+                new_comment += f", {notes=}, {reason=}, {user=}"
+                new_comment = generate_hmac_sha256_msg(key, new_comment)
+                response = client.close_incident_request(new_incident_id, new_comment)
+
+            elif (lumu_status := parsed_args.delta.get("lumustatus")) or (
+                lumu_status := parsed_args.delta.get("lumu_status")
+            ):
+                if lumu_status in ["mute", "muted"]:
+
+                    new_comment = generate_hmac_sha256_msg(key, new_comment)
+
+                    response = client.mute_incident_request(
+                        new_incident_id, new_comment
+                    )
+
+                elif lumu_status in ["unmute", "unmuted"]:
+                    new_comment = generate_hmac_sha256_msg(key, new_comment)
+                    response = client.unmute_incident_request(
+                        new_incident_id, new_comment
+                    )
+
+                elif lumu_status in ["close", "closed"]:
+                    msg = "to close an incident, it has to be in the CLOSE INCIDENT option in the ACTION button"
+                    response = {"statusCode": 404, "reason": msg}
+                    demisto.debug(msg)
+
+                elif original_commnet:
+                    new_comment = generate_hmac_sha256_msg(key, new_comment)
+                    response = client.comment_a_specific_incident_request(
+                        new_incident_id, new_comment
+                    )
+
+                else:
+                    response = {"statusCode": 404, "reason": "lumu status not found"}
+
+            elif original_commnet:
+
+                new_comment = generate_hmac_sha256_msg(key, new_comment)
+
+                response = client.comment_a_specific_incident_request(
+                    new_incident_id, new_comment
+                )
+
+        except DemistoException as err:
+            response = repr(err)
+            error_msg = "Failed to parse json object from response: b''"
+            if str(err) == error_msg:
+                response = {"statusCode": 200}
+            else:
+                response = repr(err)
+                raise DemistoException(err)
+
+    demisto.debug(
+        f"{response=}, {parsed_args.delta=}, {parsed_args.remote_incident_id=}, {parsed_args.incident_changed}"
+    )
+
+    return new_incident_id
+
+
+def clear_cache_command():
+    cache: Dict = {}
+    cache["cache"] = []
+    cache["lumu_incidentsId"] = []
+    set_integration_context(cache)
+    return f"cache cleared {get_integration_context()=}"
+
+
+def get_cache_command():
+
+    cache = get_integration_context()
+
+    # demisto.debug(f'{cache=}')
```

Comment:
```suggestion

```

Reply:
comment removed in commit 2bb633df7de2b53b93d8a8cdc1fcaef13e33c6a7